soundcard/pulseaudio.py

```diff
@@ -40,7 +40,7 @@ def get_speaker(id):
     return _Speaker(id=_match_soundcard(id, speakers)['id'])
 
 
-def all_microphones(exclude_monitors=True):
+def all_microphones(include_loopback=False):
```

Comment:
We should still allow `exclude_monitors` for the time being, in parallel with `include_loopback`, but raise a `DeprecationWarning` if it is used, and convert it to the new `include_loopback`.